### PR TITLE
archive posit::conf(2024) slides

### LINKED ISSUE #377: Archive 2024 conf slides
🏄

### PR DESCRIPTION
**Pull request by simonpcouch**
Closes #377. Similar story to #291—there are a couple additional files in `docs/` outside the noted directory that do seem like they should be there.

### CODE REVIEW

File: index.qmd [hunk 1]
```diff
@@ -110,6 +110,7 @@ Quarto files for working along [are available on GitHub](https://github.com/tidy
 
 ### English
 
+-   [August 2024](archive/2024-08-posit-conf/index.html) at posit::conf()
```
Comment (simonpcouch) [inline index.qmd:113]: For a moment, I saw the "plain HTML" kinda look (that is currently for the ConectaR slides, see below) if I don't specify `/index.html`. This doesn't click with my limited web understanding, but does the trick nonetheless.

Screenshot 2024-09-06 at 3 47 05 PM

Comment (hfrick) [reply index.qmd:113]: I'm not sure what to make of this but for posit conf 2023, it does not link to `index.html` explicitly but it looks fine: https://workshops.tidymodels.org/archive/2023-09-posit-conf/.  I'm okay with you merging with or without. I suggest we just have a look at the deployed site to make sure everything is okay there 😆

**Review by hfrick — APPROVED** [review]
Looks good, as far as I can tell!

File: index.qmd (hunk 1 above)
Comment (simonpcouch) [reply index.qmd:113]: Just get the site a moment to build and then checked—we're in the clear!

**Pull request merged**